<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fern</class>
 <widget class="QWidget" name="Fern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>938</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="prob1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>[0.01,0.85,0.07,0.07]</string>
   </property>
  </widget>
  <widget class="QLabel" name="bsplinelabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>581</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#00557f;&quot;&gt;Click &lt;/span&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#00557f;&quot;&gt;buttons&lt;/span&gt;&lt;span style=&quot; font-size:16pt; color:#00557f;&quot;&gt; with &lt;/span&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#00557f;&quot;&gt;LeftMouseButton &lt;/span&gt;&lt;span style=&quot; font-size:16pt; color:#00557f;&quot;&gt;to&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#00557f;&quot;&gt; choose probability distribution.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#00557f;&quot;&gt;Click 'Clear' to erase and paint again.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prob3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>150</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>[0.01,0.91,0.04,0.04]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prob2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>[0.33,0.33,0.33,0.01]</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>prob1</sender>
   <signal>clicked()</signal>
   <receiver>Fern</receiver>
   <slot>P1_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prob2</sender>
   <signal>clicked()</signal>
   <receiver>Fern</receiver>
   <slot>P2_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prob3</sender>
   <signal>clicked()</signal>
   <receiver>Fern</receiver>
   <slot>P3_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>622</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>P1_click()</slot>
  <slot>P2_click()</slot>
  <slot>P3_click()</slot>
 </slots>
</ui>
